### PR TITLE
Add more test

### PR DESCRIPTION
**Pull request by 3lvis**
Improved tests for https://github.com/UseSweet/SweetFoundation/pull/15

### CODE REVIEW

File: Tests/Tests.swift [hunk 1]
```diff
@@ -21,9 +21,17 @@ class Tests: XCTestCase {
 
     func testValidEmail() {
         let validEmail = "marijn@bakkenbaeck.no"
-        let invalidEmail = "invalidEmail.no"
```
Comment (elland) [inline Tests/Tests.swift:24]: The following valid email should be added:
`igor+testing@email.com`
`テスツ@email.com`
`üøçæ_123@email.com`

And the following invalid:
`invalidE@mail.n`

Comment (3lvis) [reply Tests/Tests.swift:24]: Will add soon :)

**Comment by elland** [comment]
How come tests passed? 🤔

**Comment by 3lvis** [comment]
They didn't, Travis is sending false positives. https://travis-ci.org/UseSweet/SweetFoundation/builds/182243017

**Comment by elland** [comment]
That's the kind of 💩 we need from a CI service. 😭

**Pull request merged**